llvm/test/CodeGen/RISCV/rvv/vmv.v.v-peephole.ll

```diff
@@ -194,3 +194,14 @@ define <vscale x 2 x i32> @unfoldable_mismatched_sew(<vscale x 2 x i32> %passthr
   %b = call <vscale x 2 x i32> @llvm.riscv.vmv.v.v.nxv2i32(<vscale x 2 x i32> %passthru, <vscale x 2 x i32> %a.bitcast, iXLen %avl)
   ret <vscale x 2 x i32> %b
 }
+
+define <vscale x 1 x i64> @undef_passthru(<vscale x 1 x i64> %passthru, <vscale x 1 x i64> %x, <vscale x 1 x i64> %y, iXLen %avl) {
```

Comment:
JFYI, I precommitted these tests, so make sure you rebase.

(I'd had another idea for an approach here, but after writing it decided this was fine, and I'll do a followup to generalize and simplify.)